tests/lifecycle/test_case_test.go

```diff
@@ -106,42 +89,12 @@ func (s serviceLifecycleTestCase) execute(resourceGroup string) error {
 		ResourceGroup:          resourceGroup,
 		Details:                serviceManager.GetEmptyInstanceDetails(),
 		ProvisioningParameters: s.provisioningParameters,
+		Parent:                 parent,
 	}
 
-	// Provision...
-	provisioner, err := serviceManager.GetProvisioner(plan)
-	if err != nil {
+	if _, err = s.provision(ctx, serviceManager, instance, plan); err != nil {
```

Comment:
I refactored this out into a method on serviceLifecyleTest so it could be reused within the setup function on the test itself. Since we don't export the actual Instance Details structs, I'm using the provision method to get an instance that's valid (although the instance details are all we really care about).